aboutusv1.html

```diff
@@ -0,0 +1,56 @@
+<!doctype html>
+<html> 
+<head>
+    <title>JumboCode: About</title>
```

Comment:
In the head tag, please add `meta` tags (see the file index.html on the branch JF-index), including a `<meta name="description" content="">` where you write the inner part of the content attribute - describe this page!